src/vs/workbench/contrib/terminalContrib/links/browser/terminalLinkParsing.ts

```diff
@@ -372,6 +372,28 @@ const winLocalLinkClause = '(?:(?:' + `(?:${winDrivePrefix}|${RegexPathConstants
 function detectPathsNoSuffix(line: string, os: OperatingSystem): IParsedLink[] {
 	const results: IParsedLink[] = [];
 
+	// Special handling for PowerShell prompts on Windows
+	// PowerShell prompts have the format "PS <path>" where path can contain spaces
+	if (os === OperatingSystem.Windows) {
+		const powershellMatch = line.match(/^PS\s+([^>]+)>\s*$/);
+		if (powershellMatch) {
```

Comment:
This is the wrong approach and a bandaid fix.

The proper fix is to add an entry to `fallbackMatchers` in terminalLocalLinkDetector.ts. I think the cmd prompt entry at https://github.com/microsoft/vscode/blob/769275a57550145c3c294e9a34ed7e4b26ca08af/src/vs/workbench/contrib/terminalContrib/links/browser/terminalLocalLinkDetector.ts#L55-L56 is meant to handle the PS case too?